src/Handler/CurlFactory.php

```diff
@@ -227,12 +227,13 @@ private function applyMethod(EasyHandle $easy, array &$conf)
         }
 
         $method = $easy->request->getMethod();
-        if ($method === 'PUT' || $method === 'POST') {
+        if (in_array($method, ['PUT', 'POST'], true)
```

Comment:
Option one: "If the method is PUT or POST then..."
Option two: "if the method is one of PUT or POST then..."

For me the first one is logically more simple as we want a literally logical answer to this question. That's how I read it. :)